src/modules/tools/zprobe/DeltaGridStrategy.cpp

```diff
@@ -0,0 +1,600 @@
+/*
+ This code is derived from (and mostly copied from) Johann Rocholls code at https://github.com/jcrocholl/Marlin/blob/deltabot/Marlin/Marlin_main.cpp
+ license is the same as his code.
```

Comment:
Maybe copying in that copyright message instead of making a reader go and find it would be better? What license are the other new files in this PR being added with? Some don't currently have any license explicitly stated.